src/nxdoc/nuxeo-server/installation/nuxeo-and-redis.md

```diff
@@ -195,6 +195,14 @@ The old Redis Transient Store can still be used with `nuxeo.transientstore.provi
 
 ## Clean-up
 
+It could happen that because of a Nuxeo cluster failure, Redis is keeping corrupted data. The following commands will help you restore a stable state.
```

Comment:
Suggestion for first sentence: `After a Nuxeo node crash (low-level error, machine reboot, etc.), Redis may still contain data that Nuxeo didn't have the opportunity to clean up.`